src/utils/fn/repeat.js

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 **/
 
-function repeat(fn, delay, ...rest) {
+function repeat(fn, delay, notCallImmediately, ...rest) {
```

Comment:
I would prefer to delay this refactor for the final polishing, since it is not as trivial as it seems at first glance, and I am not seeing a real improvement in the code.

Reply:
Out of curiosity, what's the criteria you're using to meassure the code improvement? maintainability? amount of lines?